AutoCollection/CorrelationContextManager.ts

```diff
@@ -244,6 +244,9 @@ export class CorrelationContextManager {
                 if (stackFrames.length > 3) {
                     if (stackFrames[2].trim().indexOf("at Error.AppInsightsAsyncCorrelatedErrorWrapper") === 0) {
                         stackFrames.splice(2, 1);
+                    } else if (stackFrames[1].trim().indexOf("at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError") === 0
```

Comment:
Was this logic previously covered by unit tests? If so, extra cases should be added for this.